inc/saiacl.h

```diff
@@ -3150,9 +3150,8 @@ typedef enum _sai_acl_entry_attr_t
     /**
      * @brief Disable ARS forwarding for a destination that can be a LAG or nexthopgroup
      *
-     * @type sai_acl_action_data_t sai_object_id_t
+     * @type sai_acl_action_data_t bool
```

Comment:
if this attribute is used in Sonic, this change will cause syncd crash 
FYI @rlhui @lguohan